.cirrus.yml

```diff
@@ -2,97 +2,68 @@ env:
   BUILD_DIR: "build"
   GCOVR_FLAGS: "--gcov-ignore-parse-errors --exclude-throw-branches --filter Common --filter Pcap --filter Packet --xml"
```

Comment:
maybe we could remove cirrus file no ?

Reply:
I think we can remove it after we migrate all jobs to GitHub Actions. In each PR I'll remove the jobs that are migrated